### PR TITLE
New article and project: Mysky

### PR DESCRIPTION
**Pull request by RPrudden**

### CODE REVIEW

File: _posts/2016-06-07-designing-mysky.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+author:     Rachel Prudden
+layout:     post
+title:      "Designing MySky"
+date:       2016-06-07
+summary:    "Combining UX and survey design"
+categories: ['reports', 'design']
+project:    'mysky'
+thumbnail: "https://s3-eu-west-1.amazonaws.com/informatics-webimages/articles/2016-06-07-designing-mysky/phone.jpg"
```
Comment (jacobtomlinson) [inline _posts/2016-06-07-designing-mysky.md]: 2.8MB is too big for a thumbnail.

File: _posts/2016-06-07-designing-mysky.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+To investigate this question, we wanted to gather some information on how people connect weather symbols to what actually happens. We decided to build an app letting people report the weather as they saw it.
+
+If we could build this app and start gathering data, it could be useful in various ways:
+
+__Delivering personalised weather forecasts__
+It should be possible to learn a person's preferences, and use present forecasts in accordance with these. If someone consistently reported heavy rain as drizzle, that's what it would report to them.
+
+__Testing out different sets of symbols__
+This could be alternative sets of weather symbols, or more esoteric things like colours or emojis.
```
Comment (jacobtomlinson) [inline _posts/2016-06-07-designing-mysky.md]: Maybe link to the definition of emoji (on wikipedia?)

File: _posts/2016-06-07-designing-mysky.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+__Feeding information into WOW (Weather Observations Website)__
```
Comment (jacobtomlinson) [inline _posts/2016-06-07-designing-mysky.md]: Link to WOW?

File: _projects/mysky.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+completed:   false
+project:     mysky
+thumbnail:   "https://s3-eu-west-1.amazonaws.com/informatics-webimages/Antoine%27s+image/my-sky-header.jpg"
+---
+
+## Project Description
+
+However detailed and accurate our forecasts, for most people it all comes down to a single symbol. We assume these symbols make sense, but does everyone interpret them in the same way?
+
+To find out, we've been building an app called MySky. Users can report the weather as they see it, by choosing an icon representing the sky above them. From this, we hope to learn more about how people interpret our forecasts, and to start experimenting with weather information tailored to a particular user.
```
Comment (jacobtomlinson) [inline _projects/mysky.md]: Bold on mysky?

**Comment by jacobtomlinson** [comment]
LGTM, see line notes for recommended changes.

File: _posts/2016-06-07-designing-mysky.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+The app collects the user's location, time of day, choice of symbol and device ID and stores these in a DynamoDB table. The location and time can be used to compare the chosen symbol with the official forecast and observations in that area. The reason for storing a device ID is to be able to learn a user's personal preferences regarding weather symbols, in a way which doesn't involve handling personal data.
+
+## Next steps
+
+We're currently working on getting MySky into the Android store. Watch this space!
+
+<img src="https://s3-eu-west-1.amazonaws.com/informatics-webimages/articles/2016-06-07-designing-mysky/phone.jpg" alt="app on phone" width="50%" align="middle">
+
+[mysky]: http://www.informaticslab.co.uk/report/2016/05/09/mysky-is-coming.html
```
Comment (jacobtomlinson) [inline _posts/2016-06-07-designing-mysky.md]: This comment originally was going to read "Please put these links alphabetically", but reading that back I sound mental. 😃

Comment (RPrudden) [inline _posts/2016-06-07-designing-mysky.md]: haha!

**Pull request merged**